src/main/java/by/exadel/internship/controller/FormController.java

```diff
@@ -55,4 +56,10 @@ public void restoreForm(@PathVariable UUID formId) {
         formService.restoreFormById(formId);
     }
 
+    @PutMapping("/updateStatus")
```

Comment:
Shouldn't this endpoint be available only for users with some permissions?

Reply:
done